telemetry/client.go

```diff
@@ -72,7 +75,29 @@ func (c *Client) PushReceivedEnvelope(envelope *v2protocol.Envelope) {
 	body, _ := json.Marshal(postBody)
 	_, err := c.httpClient.Post(url, "application/json", bytes.NewBuffer(body))
 	if err != nil {
-		c.logger.Error("Error sending envelope to telemetry server", zap.Error(err))
+		c.logger.Error("Error sending received envelope to telemetry server", zap.Error(err))
+	} else {
+		c.logger.Debug("Successfully pushed received envelope to telemetry server", zap.String("hash", envelope.Hash().String()))
+	}
+}
+
+func (c *Client) PushSentEnvelope(envelope *v2protocol.Envelope, publishMethod wakuv2.PublishMethod) {
```

Comment:
Probably good idea to consider https://github.com/status-im/status-go/issues/5240

Reply:
I have a PR to address that issue: https://github.com/status-im/status-go/pull/5251

I'm thinking it might make most sense to batch this PR, the above, and one more that adds the status version to all telemetry calls into a single PR (and do the same with corresponding changes in telemetry service)